contract/src/msgs/data_requests/execute/dr_events.rs

```diff
@@ -1,9 +1,9 @@
 use cosmwasm_std::Event;
-use seda_common::msgs::data_requests::TimeoutConfig;
+use seda_common::msgs::data_requests::DrConfig;
 
 use super::CONTRACT_VERSION;
 
-pub fn create_timeout_config_event(config: TimeoutConfig) -> Event {
+pub fn create_dr_config_event(config: DrConfig) -> Event {
     Event::new("seda-timeout-config").add_attributes([
```

Comment:
We should rename this, add the attribute, and update the indexer :)

Reply:
Renamed and new attribute added for new field! Updated pr here with a link to new issue on the indexer/explorer since I have 0 clue where that data lives/if there's any front end changes required.